src/shared/components/mutationTable/column/ClinVarColumnFormatter.tsx

```diff
@@ -29,7 +27,7 @@ export default class ClinVarColumnFormatter {
         return (
             <div data-test="clinvar-data">
                 <ClinVar
-                    className={generalStyles['integer-data']}
+                    className={''}
```

Comment:
why do we need a className here?

Reply:
Good catch. We should just update this one instead: https://github.com/cBioPortal/cbioportal-frontend/blob/573529f93b08fab807b624518f4d89402d0c9cdc/packages/react-mutation-mapper/src/component/column/ClinVar.tsx#L39